images/buildpack-builder/builder.toml

```diff
@@ -0,0 +1,13 @@
+[stack]
```

Comment:
Are there any tests that can be written for the changes in this PR?

Reply:
Added @ `BuildpackDetectCommandTest`